package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-chatjs",
-  "version": "2.3.2",
+  "version": "2.3.3",
```

Comment:
should we bump the minor version - 2.4.0?

Reply:
good idea, Can do that.